nvme-ioctl.c

```diff
@@ -393,39 +394,55 @@ int nvme_get_log(int fd, __u32 nsid, __u8 log_id, __u32 data_len, void *data)
 	__u16 numdu = numd >> 16, numdl = numd & 0xffff;
 
 	cmd.cdw10 = log_id | (numdl << 16);
+	if (lsp)
+                cmd.cdw10 |= ((int)lsp) << 8;
+
 	cmd.cdw11 = numdu;
+        cmd.cdw12 = lpo;
```

Comment:
LPO is 64-bit field separated to LPOL and LPOU.
Where is assign statement for cdw13?

Reply:
I didn't add it. I was under the impression controllers aren't going to need to index past 4gb into a log for a long time... But I guess we'll slap it in so we do not have to do it in the future.